.travis.yml

```diff
@@ -1,15 +1,20 @@
 # sudo is required in order to be able to run src/shaping tests
 sudo: required
-# trusty is required to support setting up dummy interfaces with netlink
-dist: trusty
-language: go
-go:
-    - 1.5
-# Enforce upstream path
-go_import_path: github.com/facebook/augmented-traffic-control
-install:
-    # install dependencies
-    - ./setup.sh
-    # install assets
-    - make src/api/bindata.go
+services:
+    - docker
+
+env:
+    COMPOSE_VERSION: 1.5.0
 
+before_install:
+    - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+    - chmod +x docker-compose
+    - sudo mv docker-compose /usr/local/bin
+
+script:
+    - docker-compose build atc
+    # install dependencies
+    - docker-compose run atc ./setup.sh
+    # make binaries and test
+    - docker-compose run atc make
+    - docker-compose run atc make tests
```

Comment:
First thought: `make` should run tests and build binaries, why run `make tests` too?
<went and checked>
Shit! `make` is equivalent to `make tests`. This is a problem. #193